sql/CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ DTRACE_INSTRUMENT(sql)
 TARGET_LINK_LIBRARIES(sql ${MYSQLD_STATIC_PLUGIN_LIBS} 
   mysys mysys_ssl dbug strings vio pcre ${LIBJEMALLOC}
   ${LIBWRAP} ${LIBCRYPT} ${LIBDL} ${CMAKE_THREAD_LIBS_INIT}
-  ${SSL_LIBRARIES})
+  ${LIBSYSTEMD} ${SSL_LIBRARIES})
```

Comment:
sd_notify needed libsystemd-daemon

lack of socket activation and notify puts systemd into a quite hackish workaround territory (there was some early discussion of this on one of the MDEVs).

So i'm thinking include the sd_notify into the source tree so at least the notify method can be used in system scripts.